log_example_test.go

```diff
@@ -7,8 +7,33 @@ import (
 	"time"
 
 	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
 )
 
+// Simple logging example using the log package.
+func ExampleSimple() {
+	// UNIX Time is faster and smaller than most timestamps
+	// If you set zerolog.TimeFieldFormat to an empty string,
+	// logs will write with UNIX time
+	zerolog.TimeFieldFormat = ""
+
+	log.Print("hello world")
+
+	// Produces: {"time":1516134303,"level":"debug","message":"hello world"}
```

Comment:
Are you going to give the static test time trick a try?

Reply:
Yep!

Reply:
I'll try and figure it out ASAP...  I need to move on to learn the rest of your APIs as well - let me know if this is helpful and I can continue to help fill out some of the documentation.